easybuild/easyconfigs/p/python-parasail/python-parasail-1.1.12-intel-2018a-Python-2.7.14.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = "PythonPackage"
+
+name = 'python-parasail'
+modulename = name[len('python-'):]
```

Comment:
Hmm, I understand why you did this (to avoid a clash with `parasail` itself), but I'm wondering if that's overkill...

The `-Python` in the `versionsuffix` already gives a hint that Python bindings are provided with this module?

Reply:
Ah, wait, we actually *need* a different `name`, otherwise we'll violate Lmod's "one-name-rule" and both modules can't be loaded at the same time.